Lean4Axiomatic/Rational/Impl/Fraction/Reciprocation.lean

```diff
@@ -191,27 +191,33 @@ theorem div_eqv_fraction
   a//b                 ≃ _ := eqv_refl
 
 /--
-Every fraction can be expressed as a ratio of integers.
-
-This is a trivial theorem for fractions, but there may be other rational number
-representations for which the proof is much more difficult.
-
-**Property intuition**: Formal fractions satisfy this by definition.
-
-**Proof intuition**: Use the previous result that formal fractions are
-equivalent to division of integers, and that the denominator is nonzero because
-it's positive.
--/
-def as_ratio (p : Fraction ℤ) : AsRatio p := by
+Every fraction satisfies the rational induction axiom, that is
+if a predicate holds for all fractions of the form a / b
+then it holds for all fractions.
+
+**Property intuition**: Reasonable definitions of divison on formal
+fractions should satsify this basic property.
+
+**Proof intuition**: Expand the definitions of formal fractions, division on
```

Comment:
goal of intuition:  when we read this some tieme later, this should give the key insight to help understand the goal.